src/common/constants/TodoConstants.ts

```diff
@@ -29,22 +29,38 @@ export const enum TodoValidateFields {
   EXPIRATION_DATE = "expirationDate",
 }
 
-export const enum TodoErrorMessages {
+export const enum TodoNotificationMessages {
   MAX_LENGTH = "The maximum length of the title mustn't exceed 120 characters",
   EMPTY_TITLE = "You can't to create empty todo",
   REQUIRED_FIELD = "Required field",
-  CLEAR_COMPLETED = "Successfully cleared the completed tasks",
+  DELETE_COMPLETED_TODOS = "Successfully cleared the completed tasks",
+  DELETE_TODO = "Successfully deleted the todo",
+}
+
+export const enum TodoConfirmMessages {
+  DELETE_TODO = "delete current todo",
+  DELETE_COMPLETED_TODOS = "delete completed todos",
 }
 
 export const enum TodoTimeConstants {
   ADDITION_TODO_TIME = 5,
+  RESET_ADDITION_TODO_TIME = -5,
   TIME_INTERVAL = 1,
 }
 
-export const CurrentTodoFilter = {
+export const TodoCurrentFilter = {
   ALL: "All",
   ACTIVE: "Active",
   COMPLETED: "Completed",
 };
 
-export const CurrentTodoFilterArray = Object.values(CurrentTodoFilter);
+export const TodoCurrentFilterArray = Object.values(TodoCurrentFilter);
+
+export const enum DateInfo {
+  HOURS_IN_DAY = 24,
+}
+
+export const TodoValidateData = {
+  MAX_TITLE_LENGTH: 120,
+  MIN_TITLE_LENGTH: 1,
+};
```

Comment:
I would group a few enums/const (by category, meaning, etc) if it's possible and create different files for better code readability